reference/extensions/authorization_plugins.rst

```diff
@@ -39,7 +39,8 @@ Here we can see an example of a plugin implementation.
 Auth source plugin
 +++++++++++++++++++
 This one is a Python script that receives an ``url`` as a parameter and outputs a dictionary with the credentials or
-access token. It can also return ``None`` to indicate that Conan should proceed with its normal login flow.
+access token and an optional ``headers`` dictionary to add headers to the HTTP server request.
```

Comment:
Same as above, the ``headers`` don't have the same importance or entity than the user/password, so better treat it as a particular case after presenting the main user/password/token usage